README.md

```diff
@@ -208,7 +208,7 @@ generate some of our server and our client code.
 
 [protoc]:https://wikipedia.org/wiki/Protocol_Buffers
 
-# Step 3: Compiling Protobuf
+## Step 3: Compiling Protobuf
```

Comment:
Up in the pre-requisites section, should we add a 

```
> **Warning**
```

For Mac users to let them know they'll need to use a nightly build of rust due to differences with dependencies in Linux?

e.g. the `rustix` crate had some sort of unstable feature usage on your Mac that it didn't on my Linux system.

Reply:
Ah good!